js/checkout.phtml

```diff
@@ -0,0 +1,83 @@
+<!doctype html>
+
+<html>
+  <head>
+    <title>Razorpay</title>
+    <meta name="viewport" content="user-scalable=no,width=device-width,initial-scale=1,maximum-scale=1">
+    <meta http-equiv="pragma" content="no-cache">
+    <meta http-equiv="cache-control" content="no-cache">
+    <meta http-equiv="expires" content="0">
+    <style>
+        img{max-width: 100%; height: auto;}
+        body{font-family: ubuntu,helvetica,verdana,sans-serif; font-size: 14px; text-align: center; color: #414141; padding-top: 40px; line-height: 24px;background:#fff;}
+        label{position: absolute; top: 0; left: 0; right: 0; height: 100%; line-height: 32px; padding-left: 30px;}
+        input[type=button]{
+            font-family: inherit;
+            padding: 12px 20px;
+            text-decoration: none;
+            border-radius: 2px;
+            border: 0;
+            width: 124px;
+            background: none;
+            margin: 0 5px;
+            color: #fff;
+            cursor: pointer;
+            -webkit-appearance: none;
+        }
+        input[type=button]:hover{background-image: linear-gradient(transparent,rgba(0,0,0,.05) 40%,rgba(0,0,0,.1))}
+        .grey{color: #777; margin-top: 20px; font-size: 12px; line-height: 18px;}
+        .danger{background-color: #EF6050!important}
+        .success{background-color: #61BC6D!important}
+    </style>
+    <script src="https://checkout.razorpay.com/v1/checkout.js"></script>
+    <script>
+        var data = #json#;
+
+        data.handler = function (response) {
+            document.getElementById("razorpay_id").value = response.razorpay_payment_id;
+            document.getElementById("razorpay_signature").value = response.razorpay_signature;
+            document.getElementById("razorpay").submit();
+        };
+        data.modal = {
+            ondismiss: function() {
+                window.location.href = "#error_return_url#" ; // config not a valid variable
```

Comment:
While we used this in the other place, I am deeply skeptical of this approach. We should be using https://code.tutsplus.com/tutorials/how-to-pass-php-data-and-strings-to-javascript-in-wordpress--wp-34699 instead.